packages/zoe/src/contracts/barterExchange.js

```diff
@@ -1,136 +1,141 @@
 // @ts-check
 
 import harden from '@agoric/harden';
+import makeStore from '@agoric/store';
 import { makeZoeHelpers, defaultAcceptanceMsg } from '../contractSupport';
 
-/** @typedef {import('../zoe').ContractFacet} ContractFacet */
-
 /**
+ * This barter exchange accepts offers to trade arbitrary goods for other
```

Comment:
Capitalize "barter exchange"? You do so on line 12, but it's ambiguous whether you're referring to a general b.e. here (in which case "barter exchange contract" might be clearer) or this specific contract (in which case "This contract, a barter exchange," might be clearer phrasing).